src/main/java/com/three_tech_solutions/slot_app/dto/StudentRequestDTO.java

```diff
@@ -0,0 +1,50 @@
+package com.three_tech_solutions.slot_app.dto;
+
+import jakarta.validation.constraints.*;
+import lombok.Data;
+
+import java.time.LocalDate;
+import java.util.UUID;
+
+@Data
+public class StudentRequestDTO {
+    //El nombre no puede estar vacío
+    @NotBlank
+    private String name;
+
+    //El apellido no puede estar vacío
+    @NotBlank
+    private String lastName;
+
+    //El numero de teléfono no puede estar vacío y debe tener un formato de numero de teléfono.
+    @NotBlank
+    @Pattern(regexp = "\\d{9,15}") //minimo 9 digitos numéricos y maximo 15, sin espacios
+    private String cellphoneNumber;
+
+    //El tipo de pago no puede ser nulo
+    @NotNull
+    private UUID planTypeId; //porque es el ID de la clase
```

Comment:
Como cambiamos el PlanType, ahora ya no vas a recibir el ID del tipo de plan sino el enum directamente

```suggestion
    private PlanType planType;
```